### DISCUSSION TITLE
2.2.0

### DISCUSSION THREAD

**Discussion by patrickdemooij9** [patrickdemooij9/SeoToolkit.Umbraco#133] [General]
Hi everyone!

A nice update has been released for SeoToolkit. Pushing the package to 2.2.0. There are two changes that I would like to highlight.

## Robot tags
You are now able to setup robot tags for your document types. So if you have a type of page that you don't want Google to index, you can easily set it up within Umbraco without adding additional coding. 
This setting isn't available on the content level yet, as I still have to figure out how to make that work together with fallback values.
![image](https://user-images.githubusercontent.com/11466511/198383097-01f2f6db-33bb-4268-b502-cf928d0ffdec.png)

## Site Audit
Last time, I released a beta version for Site Audit. Unfortunately, there wasn't a lot of feedback to help me with the package development, but I have been able to have it running on one of my websites since the beta and I think it is time to add it to the core package.

Site audit has the following features:
- Create your own audits.
- Control the time between crawls & the number of pages crawled.
- Choose for multiple checks (Broken links, broken images, etc)
- All audits are stored in the database, so you can come back whenever you want to compare results

And that is all for this update. There were also some bugfixes but nothing note worthy. If you find any issues or just have feedback, then please make an issue on the Github page to make this package even better!

## Full change log
* Added robot headers to document type settings by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/126
* 122: Don't show content app if there is nothing to show by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/127
* Added umbraco-marketplace tag by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/128
* Filter out non-images for open graph image by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/129
* Allow for custom open graph crop by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/130
* Allow selecting of tinymce for your meta fields by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/131
* Siteaudit feature by @patrickdemooij9 in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/pull/132

**Full Changelog**: https://github.com/patrickdemooij9/SeoToolkit.Umbraco/compare/2.1.4...2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/patrickdemooij9/SeoToolkit.Umbraco/releases/tag/2.2.0'>2.2.0</a>.</em>

**Comment by ianleeder** [comment]
Great work @patrickdemooij9.  Any update on robot tags at the content level?
